caiman/gui/gui_pyqtgraph_layout.py

```diff
@@ -46,13 +50,28 @@ def make_color_img(img, gain=255, out_type=np.uint8):
 # movie
 # mov = cm.load('/Users/agiovann/caiman_data/example_movies/memmap__d1_60_d2_80_d3_1_order_C_frames_2000_.mmap')
 mov = cm.load(cnm_obj.mmap_file)
+min_mov = np.min(mov)
+max_mov = np.max(mov)
 
 # load summary image
 # Cn = cm.load('/Users/agiovann/caiman_data/example_movies/memmap__d1_60_d2_80_d3_1_order_C_frames_2000__Cn.tif')
 Cn = cnm_obj.estimates.Cn
 
 
 estimates = cnm_obj.estimates
+min_mov_denoise = np.min(estimates.A.dot(estimates.C))
+max_mov_denoise = np.max(estimates.A.dot(estimates.C))
+min_background = []
+max_background = []
+background_num = -1
+neuron_selected = False
+nr_index = 0
+
+for i in range(0,estimates.f.shape[0]):
+    min_background.append(np.min(estimates.b[:,i].reshape(4800,1).dot(estimates.f[i,:].reshape(1,2000))))
+    max_background.append(np.max(estimates.b[:,i].reshape(4800,1).dot(estimates.f[i,:].reshape(1,2000))))
```

Comment:
You can simply replace that with
```
min_background = np.min(estimates.b, axis=0)*np.min(estimates.f, axis=1)
max_background = np.max(estimates.b, axis=0)*np.max(estimates.f, axis=1)
```
more efficient plus does not depend on the dimensions.

Reply:
Thank you for the suggestion! I fixed and pushed it